datastream-server-restli/src/test/java/com/linkedin/datastream/server/TestCoordinator.java

```diff
@@ -3613,6 +3613,8 @@ public void testThroughputViolatingTopicsHandlingForSingleDatastream() throws Ex
     String testCluster = "testThroughputViolatingTopicsHandlingForSingleDatastream";
     String connectorType = "connectorType";
     String streamName = "testThroughputViolatingTopicsHandlingForSingleDatastream";
+    String numThroughputViolatingTopicsMetric =
```

Comment:
yes only for testing purposes. I went this route since other tests have similar behavior as the metrics subclass is private. But instead I created a visible for testing function to retrieve the metric name.